agent/util-scripts/pbench-metadata-log

```diff
@@ -151,6 +151,16 @@ function metadata_log_start {
     local dir=$1
     local group=$2
     local ts=$3
+
+    # Opportunistically capture the caller's SSH configuration in case they
+    # have host specific configurations they might want to consider in the
+    # future when reviewing historical data.  The host names used in the
+    # config file might be very different from what tools capture in their
+    # output.
+    cp -L  ${HOME}/.ssh/config   ${dir}/ssh.config > /dev/null 2>&1
```

Comment:
Just a few ruminations pointing out the obvious (these are actually questions that should be added to the issue - I'll do that but I'll leave them here as well for now):

I wonder if there could be anything in .ssh/config that should be scrubbed.
Also, until we get the agent to run under a non-root user id, this is going to be the root .ssh/config.